src/components/screens/ScoreSetEditor.vue

```diff
@@ -153,55 +152,40 @@
                 </div>
                 <div class="field">
                   <span class="p-float-label">
-                    <Chips
-                        ref="doiIdentifiersInput"
-                        v-model="doiIdentifiers"
-                        :id="$scopedId('input-doiIdentifiers')"
-                        :addOnBlur="true"
-                        :allowDuplicate="false"
-                        @add="acceptNewDoiIdentifier"
-                        @keyup.escape="clearDoiIdentifierSearch"
-                      >
-                        <template #chip="slotProps">
-                          <div>
-                              <span>{{ slotProps.value.identifier }}</span>
-                          </div>
-                      </template>
-                    </Chips>
+                    <AutoComplete ref="doiIdentifiersInput" v-model="doiIdentifiers"
```

Comment:
it looks like when you resolved conflicts, it reverted a bunch of code that was previously changed. you may need to do it again

Reply:
Yes, it reverted the codes back. TAT